build/templates/installation.inc.mako

```diff
@@ -0,0 +1,28 @@
+<%
+    import build.helper as helper
+
+    config        = template_parameters['metadata'].config
+    attributes    = helper.filter_codegen_attributes(config['attributes'])
+    functions     = helper.filter_codegen_functions(config['functions'])
+    module_name   = config['module_name']
+    driver_name   = config['driver_name']
+    c_function_prefix = config['c_function_prefix']
+%>\
+
+.. _${module_name}_installation-section:
+
+Installation
+------------
+
+As a prerequisite to using the ${module_name} module, you must install the ${driver_name} runtime on your system. Visit `ni.com/downloads <http://www.ni.com/downloads/>`_ to download the driver runtime for your devices.
```

Comment:
Since we are using mako, we could mention the specific driver and even add a link to the installer download (although I can see why we wouldn't want to add that last piece of maintenance burden).

Reply:
This does mention the specific driver. For NI-DMM this becomes:
As a prerequisite to using the nidmm module, you must install the NI-DMM runtime on your system.